pkg/operator/podidentity/podidentitywebhook_controller.go

```diff
@@ -293,8 +318,8 @@ func (r *staticResourceReconciler) ReconcileResources(ctx context.Context) error
 		r.logger.Infof("%s reconciled successfully", result.Type)
 	}
 
-	// "v4.1.0/aws-pod-identity-webhook/deployment.yaml"
-	requestedDeployment := resourceread.ReadDeploymentV1OrDie(v410_00_assets.MustAsset("v4.1.0/aws-pod-identity-webhook/deployment.yaml"))
+	// "v4.1.0/<platform_type_files>/deployment.yaml"
```

Comment:
If you are closing over the source of the deployment manifest, you can't assume paths like this outside of the implementations. Alternatively, consider removing the interface entirely and just building these paths dynamically. Not sure the interface adds a huge amount of value as it's written today.

Reply:
Actually i prefer to remove the comments, i assume interface can be helpful in future when more changes will be between webhooks